ui/tests/unit/adapters/node-test.js

```diff
@@ -86,157 +93,146 @@ module('Unit | Adapter | Node', function(hooks) {
     );
   });
 
-  test('setEligible makes the correct POST request to /:node_id/eligibility', async function(assert) {
-    const { pretender } = this.server;
-    const node = await run(() => this.store.findRecord('node', 'node-1'));
-
-    await this.subject().setEligible(node);
-
-    const request = pretender.handledRequests.lastObject;
-    assert.equal(
-      request.url,
-      `/v1/node/${node.id}/eligibility`,
-      'Request was made to /:node_id/eligibility'
-    );
-    assert.equal(request.method, 'POST', 'Request was made with the POST method');
-    assert.deepEqual(
-      JSON.parse(request.requestBody),
-      {
+  const testCases = [
+    {
+      variation: '',
+      id: 'node-1',
+      region: null,
+      eligibility: 'POST /v1/node/node-1/eligibility',
+      drain: 'POST /v1/node/node-1/drain',
+    },
+    {
+      variation: 'with non-default region',
+      id: 'node-1',
+      region: 'region-2',
+      eligibility: 'POST /v1/node/node-1/eligibility?region=region-2',
+      drain: 'POST /v1/node/node-1/drain?region=region-2',
+    },
+  ];
+
+  testCases.forEach(testCase => {
+    test(`setEligible makes the correct POST request to /:node_id/eligibility ${testCase.variation}`, async function(assert) {
+      const { pretender } = this.server;
+      if (testCase.region) window.localStorage.nomadActiveRegion = testCase.region;
```

Comment:
I tried this and it did not work. I think since tests can run in any order, the only way to make this work is to give each test case its own module...but I'm gonna avoid the nerd snipe 😅